src/index.css

```diff
@@ -26,6 +26,7 @@ body {
   align-items: center;
   min-width: 320px;
   min-height: 100vh;
+  background: #000;
```

Comment:
Please correct the bg-color according to design.
And remove unused styles
```
 color-scheme: light dark;
  color: rgba(255, 255, 255, .87);
  background-color: #242424;
```

Reply:
tried to do it

Reply:
done